### DISCUSSION TITLE
Regarding Model location aka cache can move to another place aside C drive ?

### DISCUSSION THREAD

**Discussion by tukangcode** [rupeshs/fastsdcpu#137] [General]
hello and how are you everyone, 
i managed install and run fastsd but the problem is location of sd model are save and set at .cache at windows system and my space at c is very running low space, is there way to change location of model ? thanks and sorry for bad english

**Comment by rupeshs** [comment]
@tukangcode Thanks for using FastSD CPU, could you please check this discussion https://github.com/rupeshs/fastsdcpu/discussions/102

**Reply by rupeshs** [reply]
To change the cache path set **HF_HOME** environment variable.
For Winodws `set HF_HOME=D:/mycache`
For Linux/Mac `export HF_HOME=/home/mycache`
Also, you can add this to start.bat or start.sh file.
This should work!

**Comment by tukangcode** [comment]
Ok will try at morning later here already very late night thanks for help

Pada Sel, 13 Feb 2024 23.46, Rupesh Sreeraman ***@***.***>
menulis:

**Comment by tukangcode** [comment]
it works thanks !
